### ISSUE TITLE
[Suggestion] IOS Alert Style for Android

### ISSUE THREAD

**Issue by AdzeB** [nandorojo/burnt#35]
Hey, I just wanted to put a suggestion out there: Could you implement the IOS Alert style for Android instead of using toast styles that come from the button? The suggestion has been implemented for [Ting](https://github.com/baronha/ting).

**Comment by nandorojo** [comment]
Yeah I’m open to it. Ting looks like a pretty big ripoff of Burnt without credit.

**Comment by AdzeB** [comment]
To be fair, He did state on the Reddit post(where I found the library) that the library takes inspiration from Burnt. He also refers to Burnt for Expo Users on the GitHub project.

**Comment by nandorojo** [comment]
Gotcha, I didn’t see the expo mention in the docs since it was a bit hidden. I’d expect it to be a bit more prominent on the docs and link back to Burnt since it’s using the same upstream libraries and essentially copying the API.

In any case, I’m curious what the best Android library is

**Comment by AdzeB** [comment]
Oh I see, are you still thinking of implementing [devxoul/Toaster#appearance](https://github.com/devxoul/Toaster#appearance)?

**Comment by sidorchukandrew** [comment]
I think a good choice would be to try porting over Material 3's snackbar. The [Android docs](https://developer.android.com/guide/topics/ui/notifiers/toasts#alternatives_to_using_toasts) seem to suggest that a snackbar would be better suited, especially since these are displayed in the foreground. @nandorojo Would it be helpful for me to create a PR?

**Comment by nandorojo** [comment]
yeah that should work as a step up from ToastAndroid

**Comment by sidorchukandrew** [comment]
I just found out that `react-native-snackbar` uses Google's native Snackbar component on Android ([see here](https://github.com/cooperka/react-native-snackbar/blob/8612f4c9cf6079b753bdd99e59ff00c66031586c/android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java#L154)). Maybe the Android side here could pull that in.
